### PR TITLE
feat: faq application is setup #32

### PR DESCRIPTION
**Pull request by n1rjal**
faq application is setup with the following
1. Models
2. Serializer
3. Admin Panel
4. views
5. migrations
6. Swagger docs

image

image

### CODE REVIEW

File: faq/models.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Create your models here.
+class Faq(models.Model):
+    question = models.CharField(max_length=700, null=False, blank=False)
+    answer = models.CharField(max_length=700, null=False, blank=False)
+    position = models.IntegerField(
+        null=False,
+        blank=False,
+        validators=[MinValueValidator(1)],
+    )
```
Comment (rex9840) [inline faq/models.py:13]: will it  better to add tags for  faq like github ??

File: faq/models.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+
+
+# Create your models here.
+class Faq(models.Model):
+    question = models.CharField(max_length=700, null=False, blank=False)
+    answer = models.CharField(max_length=700, null=False, blank=False)
+    position = models.IntegerField(
```
Comment (rex9840) [inline faq/models.py:9]: what does position do in the model???

**Review by rex9840 — CHANGES_REQUESTED** [review]

File: faq/models.py (hunk 1 above)
Comment (n1rjal) [reply faq/models.py:13]: No. We have FAQ already in the website

File: faq/models.py (hunk 2 above)
Comment (n1rjal) [reply faq/models.py:9]: 1 means fisrt, 2 means second , etc

**Review by Sailesh-Singh — DISMISSED** [review]

**Review by rex9840 — DISMISSED** [review]

**Review by Sailesh-Singh — APPROVED** [review]

**Review by rex9840 — APPROVED** [review]

**Pull request merged**